tests/testthat/test-unnecessary_placeholder_linter.R

```diff
@@ -14,8 +14,8 @@ patrick::with_parameters_test_that(
     # . used inside a scope --> ok
     expect_lint(sprintf("x %s { foo(arg = .) }", pipe), NULL, linter)
   },
-  .test_name = c("forward", "assignment", "tee"),
-  pipe = c("%>%", "%<>%", "%T>%")
+  .test_name = c("forward", "assignment", "tee", "greedy", "extraction"),
```

Comment:
Test names could be made consistent with the other tests.

Reply:
Shared logic in helpers.R now, PTAL

Reply:
looks good